include/flyft/mesh.h

```diff
@@ -34,17 +37,28 @@ class Mesh
         int asShape(double dx) const;
 
         double asLength(int shape) const;
+        
+        double integrateSurface(int idx, double j_lo, double j_hi) const;
+        
+        double integrateSurface(int idx, const DataView<double>& j) const;
+        
+        double integrateVolume(int idx, double f) const;
 
+        double integrateVolume(int idx, const DataView<double>& f) const;
```

Comment:
For readability, group overloaded functions together
```suggestion
        double integrateSurface(int idx, double j_lo, double j_hi) const;
        double integrateSurface(int idx, const DataView<double>& j) const;
        
        double integrateVolume(int idx, double f) const;
        double integrateVolume(int idx, const DataView<double>& f) const;
```